sys-apps/smartmontools/smartmontools-6.4.ebuild

```diff
@@ -49,9 +49,9 @@ src_configure() {
 	econf \
 		--docdir="${EPREFIX}/usr/share/doc/${PF}" \
 		--with-initscriptdir="${EPREFIX}/etc/init.d" \
-		"$(use_with caps libcap-ng)" \
```

Comment:
you added these errors in a previous commit.  please revert/squash so these are never a problem.